### PR TITLE
Disable CI benchmark suite

### PR DESCRIPTION
**Pull request by pepeiborra**
It appears that the benchmark suite gets stuck due to `stack exec` grabbing a file lock that it didn't use to. 

This PR is to debug the issues to try find a fix or workaround

```

ghcide> still blocking for directory lock on /home/vsts/work/1/s/.stack-work/dist/x86_64-linux/Cabal-2.4.0.1/build-lock; maybe another Stack process is running?
ghcide> still blocking for directory lock on /home/vsts/work/1/s/.stack-work/dist/x86_64-linux/Cabal-2.4.0.1/build-lock; maybe another Stack process is running?
ghcide> still blocking for directory lock on /home/vsts/work/1/s/.stack-work/dist/x86_64-linux/Cabal-2.4.0.1/build-lock; maybe another Stack process is running?
```

### CODE REVIEW

**Comment by pepeiborra** [comment]
Updated to disable the bench CI script until I (or someone else) can find the time to unstuck it

File: bench/hist/Main.hs [hunk 1]
```diff
@@ -290,11 +290,11 @@ buildGhcide Stack out =
...
 
-findGhc :: BuildSystem -> Action FilePath
-findGhc Cabal =
+findGhc :: FilePath -> BuildSystem -> Action FilePath
+findGhc _cwd Cabal =
     liftIO $ fromMaybe (error "ghc is not in the PATH") <$> findExecutable "ghc"
-findGhc Stack = do
-    Stdout ghcLoc <- cmd (s "stack exec which ghc")
+findGhc cwd Stack = do
+    Stdout ghcLoc <- cmd [Cwd cwd] (s "stack exec which ghc")
```
Comment (ndmitchell) [inline bench/hist/Main.hs:297]: Should we undo this change? Or is it worth keeping even though it didn't work out?

Comment (pepeiborra) [reply bench/hist/Main.hs:297]: We should keep it. it's a correctness change, as the `ghc` binary used by stack is context dependent

**Review by ndmitchell — APPROVED** [review]
